File: qc-application/src/main/java/com/eager/questioncloud/application/api/user/point/UserPointService.java
package com.eager.questioncloud.application.api.user.point;

import com.eager.questioncloud.core.domain.point.infrastructure.UserPointRepository;
import com.eager.questioncloud.core.domain.point.model.UserPoint;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@Service
@RequiredArgsConstructor
public class UserPointService {
    private final UserPointRepository userPointRepository;

    public UserPoint getUserPoint(Long userId) {
        return userPointRepository.getUserPoint(userId);
    }
}

File: qc-application/src/main/java/com/eager/questioncloud/application/business/point/service/UserPointService.kt
package com.eager.questioncloud.application.business.point.service

import com.eager.questioncloud.core.domain.point.infrastructure.repository.UserPointRepository
import com.eager.questioncloud.core.domain.point.model.UserPoint
import org.springframework.stereotype.Service

@Service
class UserPointService(
    private val userPointRepository: UserPointRepository
) {
    fun getUserPoint(userId: Long): UserPoint {
        return userPointRepository.getUserPoint(userId)
    }
}
